src/Xamarin.Android.Build.Tasks/Tasks/BuildArchive.cs

```diff
@@ -43,8 +46,10 @@ public class BuildArchive : AndroidTask
 
 	public override bool RunTask ()
 	{
+		var is_aab = string.Compare (AndroidPackageFormat, "aab", true) == 0;
```

Comment:
If someone has `$(AndroidPackageFormats)=aab;apk`, does the right value come through here?

I think this is the default value for `Release` mode.

Reply:
`AndroidPackageFormat` gets calculated from `AndroidPackageFormats`.